app/scripts/lib/app-start.js

```diff
@@ -288,7 +304,8 @@ function (
             relier: this._relier,
             assertionLibrary: this._assertionLibrary,
             oAuthClient: this._oAuthClient,
-            session: Session
+            session: Session,
+            channel: this._iframeChannel
```

Comment:
If a channel is going to be passed in, might as well remove [the duplicate logic](https://github.com/mozilla/fxa-content-server/blob/14cbf41d78d918382fc414d67abc0a4fa6d81619/app/scripts/models/auth_brokers/iframe.js#L68) in the IFrameBroker.